intake_geopandas/geopandas.py

```diff
@@ -88,6 +133,19 @@ class GeoJSONSource(GeoPandasFileSource):
 class ShapefileSource(GeoPandasFileSource):
     name = "shapefile"
 
+    def _resolve_single_file(self, filelist):
+        """
+        Given a list of fsspec OpenFiles, find a .shp file.
+        """
+        local_files = fsspec.open_local(self.urlpath, **self.storage_options)
+        for f in local_files:
+            if f.endswith(".shp"):
+                return f
+        raise ValueError(
```

Comment:
```suggestion
        else 
            raise ValueError(
```

Reply:
Not incorrect to do this, but also shouldn't be necessary, since the only way to get here is if it finishes the for loop.